src/main/java/kmql/Database.java

```diff
@@ -137,6 +150,37 @@ public boolean tableInitialized(String name) {
         return getTable(name).initialized;
     }
 
+    /**
+     * Return tables contained in this database
+     * @return the set of {@link Table}
+     */
+    public Set<Table> tables() {
+        return tables.values().stream().map(m -> m.table).collect(Collectors.toSet());
+    }
+
+    /**
+     * Return columns of the given table.
+     * @param table the target table.
+     * @return the list of column names.
+     */
+    public List<String> columns(Table table) {
```

Comment:
Any points to take a `Table` instead of its `name`? 
Also, I think it's better to check the presence of the given name at `tables` map (and throw if not).